### PR TITLE
Refactor PageInfo and Search, Add Associated Passing Tests

### LINKED ISSUE #91: Refactor PageInfo

### PR DESCRIPTION
**Pull request by lindsaywparker**
## What's this PR do?

Refactor code, take out some hefty switch cases.

## Pre-Merge TODOs
 - [X] Tests Added
   - ANY change that can be tested must include testing
 - [ ] Add follow up issues?
 - [ ] 2 PR reviews completed

## What gif best describes this PR or how it makes you feel?

Closes #91

### CODE REVIEW

**Comment by lindsaywparker** [comment]
@the-oem

File: app/components/FavoriteInfo.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { DISPLAY_NAME } from '../helpers/constants';
+import { calculateStyle } from '../helpers/favUtils';
+
+const FavoriteInfo = ({ images }) => {
+  const preferenceResults = images ? calculateStyle(images) : [];
+
+  return (
+    <div className='page-info'>
+      <h2 className='page-title'>FAVORITES</h2>
+      {images ? preferenceResults.map((result, i) =>
```
Comment (the-oem) [inline app/components/FavoriteInfo.js:11]: Clever use of a ternary directly within the return statement. I hadn't thought of that but now I may need to go implement that in a few places.

**Comment by lindsaywparker** [comment]
@dstock48 @tman22

File: app/components/ImagesList.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import Image from './Image';
 
-const ImagesList = ({ isLoading, images, favorites, handleFavorite }) => {
-  const representativeImages =
-  // isLoading ?
-  //   <img className='loader' src='../assets/loader.gif' alt='Loading...' />
-  //   :
+const ImagesList = ({ images, favorites, handleFavorite }) => {
+  const representativeImages = images ?
```
Comment (dstock48) [inline app/components/ImagesList.js:5]: I like the use of the ternary here to display your loader if no images have been received by the component yet. And the fact that it eliminates the need for the `isLoading` prop! 🤘🏼

**Review by dstock48 — APPROVED** [review]

**Pull request merged**